emails/unifiedBookingEmail.hbs

```diff
@@ -13,7 +13,7 @@
             <img src="https://i.imgur.com/SkOGGbd.png" alt="Company Logo">
         </div>
         <div class="content" >
-            <h1>Bookings {{bookingStatus}}</h1>
+            <h1>Bookings <span style="color: blue;">{{bookingStatus}}</span></h1>
```

Comment:
I feel we should highlight only in the table and not here as the individual statuses are what need to stand out.